From: Jan Engelhardt <jengelh@inai.de>
Date: 2019-08-26 10:33:27.198383143 +0200


openSUSE uses -Werror=return-type since a while... :-/

Returning a value in a void function has no practical issues, unlike returning
no value in an int function. But gcc does not differentiate between the two
cases.

[  139s] send.c:4200:12: error: 'return' with a value, in function returning void [-Werror=return-type]
[  139s]  4200 |     return 0;
---
 pith/send.c |    1 -
 1 file changed, 1 deletion(-)

Index: alpine-2.21/pith/send.c
===================================================================
--- alpine-2.21.orig/pith/send.c
+++ alpine-2.21/pith/send.c
@@ -4197,7 +4197,6 @@ set_parameter(PARAMETER **param, char *p
 	      pm->value = cpystr(new_value);
 	}
     }
-    return 0;
 }
 
 
